src/utils/Component.ts

```diff
@@ -0,0 +1,192 @@
+import EventBus from "./EventBus";
+
+type TProps = {
+  [key: string]: unknown
+  events?: Record<string, (e) => void>
+}
+
+type TMeta = {
+  tagName: string,
+  attributes: Record<string, string>,
```

Comment:
А как быть, если я захочу присвоить аттрибут width, у которого значением является цифра?
```suggestion
  attributes: Record<string, string | number>,
```